src/camera_session.h

```diff
@@ -33,6 +33,7 @@ class CameraSession {
   ~CameraSession();
 
   int init();
+  int validateConfig();
```

Comment:
This can be private right?

Reply:
Yeah, we should be fine to make this private